src/inverted-index.js

```diff
@@ -0,0 +1,128 @@
+/* eslint no-unused-vars: "off"*/
+
+/**
+ * An inverted index class.
+ * @class
+ */
+class InvertedIndex {
+  /**
+   * @constructor
+   */
+  constructor() {
+    // stores all created index
+    this.index = {};
+  }
+
+  /**
+   * A token method
+   * @param {Array} words
+   * @returns {Array} Returns array
+   */
+  static tokenize(words) {
+    // exclude start and end white-space
+    return words.trim().replace(/[^a-zA-Z 0-9]+/g, '').split(/\s/);
+  }
+
+  /**
+   * A method to filter for unique words
+   * @param {Array} words
+   * @returns {Array} Returns array
+   */
+  static distinctWords(words) {
+    const tokens = InvertedIndex.tokenize(words);
+    // remove multile words
+    return tokens.filter((item, index) =>
+      tokens.indexOf(item) === index);
+  }
+
+  /**
+   * A method to create index
+   * @param {Object} fileName
+   * @param {Object} docToIndex
+   * @returns {Object} Returns object containing index
+   */
+  createIndex(fileName, docToIndex) {
+    const newindex = {};
+    const wordsToIndex = [];
+    docToIndex.forEach((document) => {
```

Comment:
## Nitpick
Conventionally, it's ideal to space out logical blocks. Example: Add a single line separation between variable declarations and logic.